dev-packages/browser-integration-tests/suites/tracing/browserTracingIntegration/pageload-update-txn-name/test.ts

```diff
@@ -10,27 +10,34 @@ import {
 import { sentryTest } from '../../../../utils/fixtures';
 import { getFirstSentryEnvelopeRequest, shouldSkipTracingTest } from '../../../../utils/helpers';
 
-sentryTest('sets the source to custom when updating the transaction name', async ({ getLocalTestPath, page }) => {
-  if (shouldSkipTracingTest()) {
-    sentryTest.skip();
-  }
+sentryTest(
```

Comment:
is this PR supposed to include changes to browser tracing (tests)? 😅

Reply:
yes. We have to export `updateSpanName` also from browser for people who'd use this funciton in code that runs both on browser and on server. So I added an integration test for it. This test, I adjusted. The changes look scarier in the diff than they actually are. Just updated the name and added some additional assertions.